src/js/pre-need/components/IntroductionPage.jsx

```diff
@@ -24,7 +24,7 @@ class IntroductionPage extends React.Component {
 
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply online for pre-need determination of eligibility in a VA National Cemetery"/>
+        <FormTitle title="Apply online for pre-need determination of eligibility in a VA national cemetery"/>
```

Comment:
![localhost-3001-burials-and-memorials-burial-planning-application-](https://user-images.githubusercontent.com/16051172/32812724-569f60be-c95b-11e7-8532-51244f0187f1.png)